translations/alve.yml

```diff
@@ -26,3 +26,4 @@ translations:
     write_review_for: 'Write a review for'
     and_help: 'and help other users!'
     review_this: 'Review this product'
+    verified_purchase: 'Verified purchase'
```

Comment:
[minor++] You missed the joy of putting a newline at the end of file.

Also in `translations/skroutz.yml`

Reply:
It has a new line now!